source/user-manual/user-administration/ldap.rst

```diff
@@ -233,20 +233,10 @@ Follow these steps to create a new role mapping and grant administrator permissi
       Done with success
       SUCC: Expected 1 config types for node {"updated_config_types":["rolesmapping"],"updated_config_size":1,"message":null} is 1 (["rolesmapping"]) due to: null
 
-#. Check the value of ``run_as`` in the ``/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml`` configuration file. If ``run_as`` is set to ``false``, proceed to the next step.
+#. Check the value of **Run as** in the API host entry configuration on **Dashboard management** > **Server APIs**. If **Run as** is set to ``false``, proceed to the next step. If **Run as** is set to ``true``, you need to add a role mapping on the Wazuh dashboard. To map the backend role to Wazuh, follow these steps:
 
-   .. code-block:: yaml
-      :emphasize-lines: 7
-
-      hosts:
-        - default:
-            url: https://localhost
-            port: 55000
-            username: wazuh-wui
-            password: "<wazuh-wui-password>"
-            run_as: false
-
-   If ``run_as`` is set to ``true``, you need to add a role mapping on the Wazuh dashboard. To map the backend role to Wazuh, follow these steps:
+   .. image:: /images/wazuh-dashboard/api-connections/wazuh-dashboard-check-api-connection-run-as.jpg
```

Comment:
I'm not sure which image is more appropriate to show here, if the `Run as` is disabled or enabled. What do you think about this @javimed ?

Reply:
Sorry I overlooked this question, @Desvelao . We could put the `Run as` is _disabled_ just like the code block we currently have showing `false`. The image would go after the "_If Run as is set to false..._" text but before the "_If Run as is set to true..._" text just like the current code block we have. I think run as false is the simplest configuration option and worth displaying.

Reply:
The displayed value as `Run as` in the API connections table is not the same value ( `false` / `true` ) that is configured in the API hosts. The raw value of the `Run as` setting is not displayed. Instead, the displayed value can display a missconfiguration.

CC: @asteriscos